### DISCUSSION TITLE
【咨询】Mac 上通过运行python3 web_demo_streamlit.py 会把模型下载到哪里？

### DISCUSSION THREAD

**Discussion by myEchos** [zai-org/ChatGLM3#1196] [Q&A] [answered]
我按照README.MD的步骤顺序往下执行，但是没有注意到我的Mac RAM 是不足以运行 chatglm 的。
但是此时已经完成了模型下载（7个 model-00001-of-00007.safetensors 大文件）。
运行过程中发现推理太慢（设备不符合条件），于是想要卸载，但是未能找到删除模型的方法，也未能找到模型所在的位置。

我做了一些尝试，但失败了：

尝试修改 “cli_demo.py”，使之在命令行输出
print(f"MODEL_PATH: {MODEL_PATH}")
print(f"TOKENIZER_PATH: {TOKENIZER_PATH}, os.path.abspath(TOKENIZER_PATH): {os.path.abspath(TOKENIZER_PATH)}")
但是得到的路径，里面并没有大文件。

通过 Mac 自带的存储文件清理，似乎我可控的文件夹内并没有大文件（难道是被统计到了“系统数据”中？）

尝试搜索，包括“THUDM/chatglm3-6b”、‘safetensors’，同样未能找到大文件。

希望能够得到各位的指导和帮助，先谢谢啦。

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
～/.cache/huggingface/ 

建议放到指定的地方，然后将地址修改为绝对路径，如果电脑配置太低，需要升级配置，在技术文档中有提到推荐的配置
